Explorer/Assets/Scripts/ECS/SceneLifeCycle/Systems/UnloadSceneSystem.cs

```diff
@@ -44,9 +46,11 @@ public void FinalizeComponents(in Query query)
         [All(typeof(DeleteEntityIntention)), None(typeof(PortableExperienceComponent))]
         private void UnloadLoadedScene(in Entity entity, ref SceneDefinitionComponent definitionComponent, ref ISceneFacade sceneFacade)
         {
-            // Keep definition so it won't be downloaded again = Cache in ECS itself
             sceneFacade.DisposeSceneFacadeAndRemoveFromCache(scenesCache, definitionComponent.Parcels, sceneAssetLock);
-            World.Remove<ISceneFacade, VisualSceneState, DeleteEntityIntention>(entity);
+
+            // Keep definition so it won't be downloaded again = Cache in ECS itself
+            if (!localSceneDevelopment)
+                World.Remove<ISceneFacade, VisualSceneState, DeleteEntityIntention>(entity);
```

Comment:
Why never remove these components? Shouldnt the scene reload all of them?

Reply:
Apparently that is done to re-use the same scene entity keeping the scene facade, and other components that don't need to be reloaded.

It's true that in any other scenario that is not Local Scene Development, it doesn't make sense to re-load the Scene Entity Definition because the content will not change in runtime, but during LSD it perfectly can.